adapter/internal/oasparser/envoyconf/routes_with_clusters.go

```diff
@@ -239,6 +239,22 @@ func CreateRoutesWithClusters(mgwSwagger model.MgwSwagger, upstreamCerts map[str
 		return routes, clusters, endpoints, nil
 	}
 
+	if mgwSwagger.GetAPIType() == constants.GRAPHQL {
+		routesP, err := createRoutes(genRouteCreateParams(&mgwSwagger, nil, vHost, apiLevelBasePathProd, apiLevelClusterNameProd,
+			apiLevelClusterNameSand, nil, nil, organizationID, false))
+		if err != nil {
+			logger.LoggerXds.ErrorC(logging.ErrorDetails{
+				Message: fmt.Sprintf("Error while creating routes for GraphQL API : %s version : %s. Error: %s",
+					apiTitle, apiVersion, err.Error()),
+				Severity:  logging.MAJOR,
+				ErrorCode: 2233,
```

Comment:
Shall we update the error logs sheet with these?

Reply:
done